pkgs/misc/base16-shell-preview/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "base16_shell_preview";
+  version = "0.3.0";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0x2fbicrcqgf2dl7dqzm14dz08vjjziabaaw33wah3v9wv4rw7jq";
+  };
+
+  doCheck = false;
```

```suggestion
  # No tests
  doCheck = false;
```

Good idea to add a comment with why checks are disabled.